repository-hbase/src/main/java/hbase/load/CSVDataKVMapper.java

```diff
@@ -151,7 +153,7 @@ private void writeRow(Context context, String rowKeyStr, String[] fields) throws
                 try {
                     put.add(new KeyValue(rowKey.get(), columnFamily, columnNames[i], fields[i].getBytes()));
                 } catch (Exception e) {
-                    LOG.error("Cannot add line '{}'", fields[0], e);
+                    LOG.error("ColumnName: '{}' \n.Cannot add line '{}' with header '{}'", columnNames, fields[0], columnNames[i], e);
                     throw e;
```

Comment:
Don't think we want to fail the whole process if one record fails to write so don't throw an exception just log the error and move on unless the error is fatal